playbooks/adhoc/uninstall.yml

```diff
@@ -539,3 +562,17 @@
     # - /etc/systemd/system/haproxy.service.d
     - /etc/systemd/system/haproxy.service.d/limits.conf
     - /etc/systemd/system/haproxy.service
+
+# Remove Contrail vRouter
+- hosts: nodes
+  become: yes
+  tasks:
+  - name: Find nodes which has vRouter
+    shell: "ip addr | grep -i vhost0 | cat"
```

Comment:
Isn't there a better way to find that out?

Reply:
Yes, will use a module.